aiosmtpd/smtp.py

```diff
@@ -207,6 +263,35 @@ def smtp_QUIT(self, arg):
             self.transport.close()
 
     @asyncio.coroutine
+    def smtp_STARTTLS(self, arg):
+        log.info('===> STARTTLS')
+        if arg:
+            yield from self.push('501 Syntax: STARTTLS')
+            return
+        if (not self.tls_context) or (not _has_ssl):
```

Comment:
You don't need the parentheses here either.  Do you think it reads better though to write it

`if not (self.tls_context and _has_tls):`

?